CORPSE_deriv.py

```diff
@@ -230,6 +249,14 @@ def CORPSE_deriv(SOM,T,theta,Ndemand,Ctransfer,params,claymod=1.0):
     derivs['AMC'] = atleast_1d(dmicrobeC['AM'])
     derivs['AMN'] = atleast_1d(dmicrobeN['AM'])
 
+    derivs['Int_ECMC'] = atleast_1d(-SOM['Int_ECMC'])
+    derivs['Int_AMC'] = atleast_1d(-SOM['Int_AMC'])
+    derivs['Int_N'] = atleast_1d(Ntransfer-Nlitter)
```

Comment:
The rate constant parameter would also need to be added to the three lines above.

Reply:
I have corrected this part accordingly.